lib/utils/token.go

```diff
@@ -27,9 +28,10 @@ import (
 // from the disk if it looks like a path,
 // otherwise, treat it as a value
 func ReadToken(token string) (string, error) {
-	if !strings.HasPrefix(token, "/") {
+	if !strings.ContainsRune(token, filepath.Separator) {
```

Comment:
This will break base64 encoded tokens because they contain a `/`.

Reply:
@russjones there won't be any change in behavior compared to what we have though?

Reply:
ah, I see, I will specify more clearly

Reply:
how about supporting file://.token - this removes the ambiguity

Reply:
Sounds good to me.